spec/requests/api/v1/contacts_spec.rb

```diff
@@ -56,3 +56,23 @@
     expect(updated_contact.email).to eq(contact_attributes["email"])
   end
 end
+
+describe "DELETE /api/v1/contacts/1" do
+  it "destroys the contact in the database" do
+    contact = create(:contact)
+
+    api_delete api_contact_path(id: contact.id)
+
+    expect(response.status).to eq(204)
+    expect {
+      Contact.find(contact.id)
+    }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
This is a weird way to check for this. You're looking for a specific exception instead of just looking for something and not finding it.